AdminServer/appscale/admin/push_worker_manager.py

```diff
@@ -85,11 +85,7 @@ def update_worker(self, queue_config):
       yield self.monit_operator.reload()
     else:
       logging.info('Restarting push worker for {}'.format(self.project_id))
-      # Monit behaves unexpectedly with a restart. This hack makes it behave
-      # more smoothly.
-      yield self.monit_operator.send_command(self.monit_watch, 'stop')
-      yield gen.sleep(.5)
-      yield self.monit_operator.send_command(self.monit_watch, 'start')
+      yield self.monit_operator.send_command(self.monit_watch, 'restart')
```

Comment:
`restart` wasn't working back when our monit stop line didn't wait around for the process to stop. Now that it has a retry parameter, the period between stop and start is no longer needed.